### PR TITLE
Security collars

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Essentially adds a new tech tree for research, that delves into... collars! Shock and no shock variants!
Pacifying and not pacifying variants!

## Why It's Good For The Game

New way to keep antags, criminals, etc in line. Or maybe to torture your victim if you're an antag yourself!

## Changelog
:cl:
add: Researchable security shock collars and pacifying collars
add: Suffering for those antagonized by shitsec
add: Collar removers, that remove... collars. (Except collar ban collars!)
/:cl:

### CODE REVIEW

**Comment by BobJoga** [comment]
It compiles.

File: modular_skyrat/code/modules/research/techweb/collars.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//snowflake file to not cause merge conflicts
+/datum/techweb_node/collars
+	id = "collars"
+	display_name = "Collar Development Technology"
+	description = "New ways for security to control their prisoners."
+	prereq_ids = list("high_efficiency")
```
Comment (Trilbyspaceclone) [inline modular_skyrat/code/modules/research/techweb/collars.dm]: not eletro tech 1?

Comment (BobJoga) [reply modular_skyrat/code/modules/research/techweb/collars.dm]: why not both?

Comment (BobJoga) [reply modular_skyrat/code/modules/research/techweb/collars.dm]: now it's good

**Comment by Useroth** [comment]
So, can you even remove those after you put them on someone? Or are those permanent?

**Comment by BobJoga** [comment]
> So, can you even remove those after you put them on someone? Or are those permanent?

Permanent. I need to make some way to make them removable again, so, hold on i'll put this on wip.

**Comment by BobJoga** [comment]
> So, can you even remove those after you put them on someone? Or are those permanent?

They are removable now. Please merge.

**Comment by Useroth** [comment]
Not entirely sure about a way this easy to cuck antagonists or people in general over.

**Comment by BobJoga** [comment]
> Not entirely sure about a way this easy to cuck antagonists or people in general over.

Pretty hard to put this collar without restraining first. And personally i think it's a better alternative than borging someone (because borgs are fucking boring).

**Comment by BobJoga** [comment]
**PR's closed due to mutes caused by Kathrin Bailey. For any grievances or complaints, contact Floof Ball#0798 on discord. The PR owner will also not listen to any of your complaints unless you provide screenshots that prove you have contacted Floof Ball#0798 on discord first.**
pool

**Pull request closed without merging**